build.gradle

```diff
@@ -185,14 +185,14 @@ configure(projectsWithFlags('java')) {
     }
 
     tasks.withType(JavaCompile) { task ->
-        if (task.path.startsWith(':examples:')) {
-            // Target Java 11 for examples.
-            options.release.set(11)
+        if (task.path.startsWith(':examples:') || task.path.contains(':resilience4j')) {
```

Comment:
It is out of scope of this PR but we need a general solution to tell a module needs Java 17 to compile.
What do you think of introducing `java17` flag for it? We may set both `java` and `java17` or only `java17` to Spring Boot 3 and Resilience4J projects.

Reply:
Agreed! Let me work on this in a separate PR 😄